experiment/tor/tor.go

```diff
@@ -230,7 +239,7 @@ func (rc *resultsCollector) defaultFlexibleConnect(
 			Path:   "/tor/status-vote/current/consensus.z",
 			Scheme: "http",
 		}
-		const snapshotsize = 1 << 16 // no need to include all in report
+		const snapshotsize = 1 << 8 // no need to include all in report
```

Comment:
I chose to reduce the snapshot as we've discussed with @hellais that we wanted to compute the simhash (which has been scheduled as part of the next cycle) and since we may not be able to deflate anyway